spec/awscr-s3/client_spec.cr

```diff
@@ -279,9 +279,9 @@ module Awscr::S3
 
         expected_objects = [
           Object.new("my-image.jpg", 434_234,
-            "\"fba9dede5f27731c9771645a39863328\""),
+            "\"fba9dede5f27731c9771645a39863328\"", "2009-10-12T17:50:30.000Z"),
           Object.new("key2", 1337,
-            "\"fba9dede5f27731c9771645a39863329\""),
+            "\"fba9dede5f27731c9771645a39863329\"", "2009-10-12T17:50:30.000Z"),
```

Comment:
It's slightly confusing both of these have the same last_modifed, despite being different in the xml response. Took me a while to figure out why the tests passed until I remembered how the Object equality was implemented. I wonder if `last_modified` should be added to the `def_equals`

Reply:
Oh awesome i didn't know def_equals was a thing. I'll replace that helper i added with this